### PR TITLE
Add pagination

### PR DESCRIPTION
**Pull request by tosih**

### CODE REVIEW

File: lib/base.js [hunk 1]
```diff
@@ -52,13 +52,49 @@ class Base {
       })
   }
 
+  _getPages (body) {
```
Comment (Nathan219) [inline lib/base.js]: add a jsdoc here

just type 
```
/**
```
into webstorm, and it'll make most of it for you

File: lib/companies.js [hunk 2]
```diff
@@ -22,26 +23,18 @@ class Companies extends Base {
...
    * List all the users who belong to a particular company
    * @param {String} companyId - ID of the company we are listing users for
    * @returns {Promise<Array>} - Returns success if intercom is disabled,
    *   otherwise returns list of users
    */
   listUsers (companyId) {
+    const users = new Users(this.client, 'users')
     return this._wrap('listUsers', { id: companyId })
       .get('body')
+      .then((body) => {
```
Comment (Nathan219) [inline lib/companies.js]: For fanciness
```
.then(body => users._getPages)
```

File: lib/tags.js [hunk 3]
```diff
@@ -6,6 +6,57 @@ const Base = require('./base')
...
+   * @see https://developers.intercom.io/reference#create-and-update-tags
+   */
+  create (tag) {
+    return this.wrap('create', tag)
+  }
+
+  /**
+   * Create or update a tag on a company/user object, passes data through to intercom
+   * @param {Object} tag
+   * @param {Object} tag.id - Tag unique identifier [required]
```
Comment (Nathan219) [inline lib/tags.js]: string or Number?  not object

File: lib/tags.js [hunk 4]
```diff
@@ -6,6 +6,57 @@ const Base = require('./base')
...
+   * @param {Object} tag.name - Tag name
```
Comment (Nathan219) [inline lib/tags.js]: string

Comment (Nathan219) [reply lib/tags.js]: Webstorm picks these up when you use the method.  You can see the parameters (and their types) by pressing ⌘+P after typing the method and a (

**Review by Myztiq — APPROVED** [review]
👍

**Review by Nathan219 — APPROVED** [review]
LGTM

**Pull request merged**